backend/src/main/java/site/coduo/retrospect/controller/response/FindRetrospectByIdResponse.java

```diff
@@ -0,0 +1,16 @@
+package site.coduo.retrospect.controller.response;
+
+import java.util.List;
+
+import site.coduo.retrospect.domain.Retrospect;
+
+public record FindRetrospectByIdResponse(String pairRoomAccessCode, List<String> answers) {
+
+    public static FindRetrospectByIdResponse of(Retrospect retrospect) {
```

Comment:
```suggestion
    public static FindRetrospectByIdResponse from(final Retrospect retrospect) {
```

final 누락이 있어요! 그리고 파라미터 값이 하나니까 of 보다 from 이 맞지 않을까요?

Reply:
동의합니다! final 누락과 함께 메서드명도 변경해볼게요.